client/source/DelphiLintClient290.dproj

```diff
@@ -1,4 +1,4 @@
-﻿<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">

```

Comment:
Probably-unintended line ending changes in this file.

Reply:
And text encoding changes, apparently! Fixing up.